test/python/test_composite.py

```diff
@@ -1264,12 +1269,12 @@ def test_yaml_user_data(self):
         gas2 = ct.Solution(self.test_work_path / "h2o2-generated-user-data.yaml")
         data2 = gas2.species(2).input_data
 
-        self.assertEqual(gas2.input_data["spam"], extra["spam"])
-        self.assertEqual(gas2.input_data["eggs"], extra["eggs"])
-        self.assertEqual(data2["foo"], "bar")
-        self.assertEqual(data2["transport"]["baz"], 1234.5)
-        self.assertEqual(data2["thermo"]["something"], [False, True])
-        self.assertTrue(gas2.reaction(5).input_data["baked-beans"])
+        assert gas2.input_data["spam"] == extra["spam"]
+        assert gas2.input_data["eggs"] == extra["eggs"]
+        assert data2["foo"] == "bar"
+        assert data2["transport"]["baz"] == 1234.5
+        assert data2["thermo"]["something"] == [False, True]
+        assert gas2.reaction(5).input_data["baked-beans"]
```

Comment:
```suggestion
        assert gas2.reaction(5).input_data["baked-beans"] is True
```

Reply:
I was wondering about how explicit we would want to be with tests like that, given that assert naturally judges the expression against being true if I understand it correctly. I did change a few cases that used to have == True to just the plain assert based off some of the pytest examples I was seeing.

Reply:
In general, I think you just want `assert x`. This was a specific exception -- if you look at the preparation of this test data, the point is that this element is supposed to be the boolean `True`, not just any truthy value.